fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ComposeTabDrawerRobot.kt

```diff
@@ -287,6 +298,19 @@ class ComposeTabDrawerRobot(private val composeTestRule: HomeActivityComposeTest
             return Transition(composeTestRule)
         }
 
+        fun toggleToSyncedTabs(interact: ComposeTabDrawerRobot.() -> Unit): Transition {
+            composeTestRule.syncedTabsButton().performClick()
+            ComposeTabDrawerRobot(composeTestRule).interact()
+            return Transition(composeTestRule)
+        }
+
+        fun clickSignInToSyncButton(interact: SyncSignInRobot.() -> Unit): TabDrawerRobot.Transition {
```

Comment:
I'm assuming this was just a copy paste error. We don't want to transition from the `ComposeTabDrawerRobot` to the XML one (`TabDrawerRobot`)
```suggestion
        fun clickSignInToSyncButton(interact: SyncSignInRobot.() -> Unit): Transition {
```

Reply:
Yes, sorry for that, made the changes.
Thanks !